raiden-ts/src/raiden.ts

```diff
@@ -684,13 +706,19 @@ export class Raiden {
    *                - secret  Secret to register, a random one will be generated if missing
    *                - secrethash  Must match secret, if both provided, or else, secret must be
    *                              informed to target by other means, and reveal can't be performed
+   *                - metadata  Used to specify possible routes instead of querying PFS.
```

Comment:
Please use
```
<ul>
   <li>...</li>
...
```

because typedoc will strip off all linebreaks etc (e.g. https://lightclient.raiden.network/docs/classes/raiden.html#transfer)